packages/main/src/themes/Popover.css

```diff
@@ -1,44 +1,3 @@
-:host {
-	display: none;
-	position: fixed;
-	min-width: 6.25rem;
-	background: var(--sapGroup_ContentBackground);
-	box-shadow: var(--sapContent_Shadow2);
-	border-radius: 0.25rem;
-	min-height: 2rem;
-	box-sizing: border-box;
-}
-
-.ui5-popover-root {
-	background: inherit;
-	border-radius: inherit;
-	width: 100%;
-	height: 100%;
-	box-sizing: border-box;
-	display: flex;
-	flex-direction: column;
-	overflow: hidden;
-}
-
-@media screen and (-ms-high-contrast: active) {
-    .ui5-popover-root {
-        border: 1px solid var(--sapPageFooter_BorderColor);
-    }
-}
-
-:host([modal])::before {
-	content: "";
-	position: fixed;
-	background-color: #000000;
-	opacity: 0.6;
-	top: 0;
-	left: 0;
-	right: 0;
-	bottom: 0;
-	outline: 0 none;
-	z-index: -1;
-}
-
 .ui5-popover-arr {
```

Comment:
.ui5-popover-arr => .ui5-popover-arrow, I got quite confused before I could find out this is not array, but arrow :)